netty/src/main/java/io/grpc/netty/NettyClientHandler.java

```diff
@@ -607,7 +644,27 @@ public void onRstStreamRead(ChannelHandlerContext ctx, int streamId, long errorC
           logger.log(Level.WARNING, String.format("Received unexpected ping ack. "
               + "Expecting %d, got %d", p.payload(), ackPayload));
         }
-      } else {
+      }
+      else if (data.readLong() == 1234){     
```

Comment:
`1234` should be a constant as well, so that it can have a name, like `BDP_MEASUREMENT_PING`